code/02_calculate-measures.R

```diff
@@ -157,9 +157,11 @@ pds %<>%
 ### 7 - Save individual level file for checking ----
 pds %>% 
 write_file(path = get_mi_data_path("ldp_data", ext = "rds", test_output = test_output))
+0 # this zero stops script from running IF write_file is overwriting an existing file, re-run the section without this line and enter 1 in the console, when prompted, to overwrite file.
```

Comment:
This is a great idea. By setting the prompt for each chunk of code, it will ensure we run things in chunks to make sure everything is working properly. This works well here